### DISCUSSION TITLE
Future Ready Talent

### DISCUSSION THREAD

**Discussion by GaneshXDX** [github/india#127] [Students]
Totally Confused with Process.

**Comment by scmishra-cse** [comment]
> Totally Confused with Process.

Tell us where you are getting stuck, Definitely will be happy to help you.

**Comment by SagnikPahari** [comment]
Cannot update details for orientation. Site is not responding. Anyone else having similar issues?

**Reply by scmishra-cse** [reply]
> Cannot update details for orientation. The site is not responding. Anyone else having similar issues?

Hi, @SagnikPahari Thank you for putting your issue here with us. 
Now coming to your issue So I want to let you know that there is some issue with the chrome browser I think because In my case the same is happening with me but after logging into Future-ready talent site with Microsoft Edge it worked fine for me and everything becomes smooth. So I'II suggests you go through it still if you face any issue in updating your details please let me know here!
